src/node_snapshotable.cc

```diff
@@ -846,9 +846,20 @@ void SnapshotBuilder::InitializeIsolateParams(const SnapshotData* data,
       const_cast<v8::StartupData*>(&(data->v8_snapshot_blob_data));
 }
 
-ExitCode SnapshotBuilder::Generate(SnapshotData* out,
-                                   const std::vector<std::string> args,
-                                   const std::vector<std::string> exec_args) {
+ExitCode SnapshotBuilder::Generate(
+    SnapshotData* out,
+    const std::vector<std::string> args,
+    const std::vector<std::string> exec_args,
```

Comment:
Could these be `const std::vector<std::string>&` ?

Reply:
Good catch. fixed.